src/scala/com/github/johnynek/bazel_deps/DepsModel.scala

```diff
@@ -464,8 +464,8 @@ sealed abstract class Language {
 }
 
 object Language {
-  case object Java extends Language {
-    def asString = "java"
+  trait JavaLike {
```

Comment:
I left `extends Language` off here to avoid
```
src/scala/com/github/johnynek/bazel_deps/Writer.scala:253: warning: match may not be exhaustive.
It would fail on the following input: JavaLike()
          lang match {
```
Might just be my lack of scala knowledge.

Reply:
If you make `sealed JavaLike extends Language ` it will work.